kafka_consumer/datadog_checks/kafka_consumer/legacy_0_10_2.py

```diff
@@ -75,7 +75,6 @@ def check(self, instance):
         topics = defaultdict(set)
 
         kafka_consumer_offsets = None
-        self._kafka_client._maybe_refresh_metadata()
```

Comment:
It makes no sense to me to have this here, since it's just checking if the TTL has expired, it doesn't even reset the TTL...